tests/testthat/test-dplyr.R

```diff
@@ -1,35 +1,73 @@
 context("Using dplyr verbs on skim objects works as expected")
+# dplyr version 0.8.0 produces a different set of classes than 
+# prior versions.  This set of tests should assure that R 
+# instances with either older or newer dplyr versions work in
+# the same way.
 
 skimmed_iris <- skim(iris)
+nvar_iris <- length(iris)
+skimmers_used_iris <- attr(skimmed_iris, "skimmers_used")
+ntypes <- length(skimmers_used_iris)
+skimmers_iris <- union(skimmers_used_iris[[1]], skimmers_used_iris[[2]])
+length_skimmed_iris <- length(skimmers_iris) + 2
+classes_skimmed_iris <- class(skimmed_iris)
 
-test_that("dplyr::filter works as expected", {
```

Comment:
I don't think you should revert the changes in the test.

Our issues with dplyr primarily emerge when we call `partition`. This happens when we print. The shape of the df isn't nearly as important and getting a printed result.

Reply:
Actually the shape a the key thing.  Filter takes a lot of forms, in one case you can filter based on `type`  and another you can filter based on a value of one of the calculated statistics and those have very different impacts on the printing.

Reply:
I haven't seen an example of where filter modifies the columns and the rows. Do you have one?

Reply:
Filter doesn't modify columns, just rows.  But it changes the print a lot if you filter based on type.